src/main/java/se/sundsvall/supportmanagement/integration/db/model/NamespaceConfigValueEmbeddable.java

```diff
@@ -0,0 +1,85 @@
+package se.sundsvall.supportmanagement.integration.db.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Embeddable;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import java.util.Objects;
+import se.sundsvall.supportmanagement.integration.db.model.enums.ValueType;
+
+@Embeddable
+public class NamespaceConfigValueEmbeddable {
+
+	@Column(name = "`key`", nullable = false)
+	private String key;
+
+	@Column(name = "`value`", columnDefinition = "text", nullable = false)
+	private String value;
+
+	@Enumerated(EnumType.STRING)
```

Comment:
This doesn't seems to work (I suppose that you want strings in DB?)

Why not use a Converter instead?

```
@Converter(autoApply = true)
public class JockesConverter implements AttributeConverter<ValueType, String> {

	@Override
	public String convertToDatabaseColumn(ValueType attribute) {
		return Optional.ofNullable(attribute).map(ValueType::toString).orElse(null);
	}

	@Override
	public ValueType convertToEntityAttribute(String dbData) {
		return ValueType.valueOf(dbData);
	}
}
```